src/DummyScreen.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import { Text, View } from 'react-native';
+
+import styles from './styles/dummy';
+
+class DummyScreen extends Component {
+  static navigationOptions = {
```

Comment:
You could define this options into a constant, to avoid defining this in every component and have each title in a single file.